src/global.scss

```diff
@@ -31,7 +31,7 @@
 // Typography
 
 .font--green {
-  color: govuk-colour('green');
+  color: govuk-colour("green");
```

Comment:
it would be good if we could use the sass variables given by for "green" or "white" so we don't hardcode the values.

https://design-system.service.gov.uk/styles/colour/
(Colour palette section)